### ISSUE TITLE
body1 is depreciated.

### ISSUE THREAD

**Issue by levimake** [londonappbrewery/bmi-calculator-flutter#9]
The TextTheme constructors from material design 2014 seems depreciated and new constructors could be used. Check them here : https://api.flutter.dev/flutter/material/TextTheme-class.html

`bodyText1` is expected instead of `body1`

`textTheme: TextTheme(body1: TextStyle(color: Colors.white)`

needs to be changed from

`textTheme: TextTheme(bodyText2: TextStyle(color: Colors.white)`

Thanks.
Not a big issue. 
Just to keep the course updated.
I thank Angela for making me looking up the docs and fixing the minor bugs.

**Comment by victorbjorklund** [comment]
Note that body1 becomes bodyText2 and body2 becomes bodyText2. I missed that at first.

**Comment by coletoncodes** [comment]
> Note that body1 becomes bodyText2 and body2 becomes bodyText2. I missed that at first.

Yup, I missed it too. The code should read: `textTheme: TextTheme(
          bodyText2: TextStyle(color: Colors.white),
        ),`
